framework/test/jest/unit/specs/controller/channels/child_process_channel.spec.js

```diff
@@ -431,7 +440,7 @@ describe('ChildProcessChannel Channel', () => {
 	describe('#_rejectWhenTimeout', () => {
 		beforeEach(() => childProcessChannel.registerToBus(socketsPath));
 
-		it('should reject with an Error object with proper message', () =>
+		it('should reject with an Error object with proper message', async () =>
```

Comment:
same above.

Reply:
removed async as it's a one-liner with implicit return